proto/cosmos/distribution/v1beta1/distribution.proto

```diff
@@ -152,3 +152,15 @@ message DelegationDelegatorReward {
     (gogoproto.nullable)     = false
   ];
 }
+
+// CommunityPoolSpendProposalJSON defines a CommunityPoolSpendProposal with a deposit
```

Comment:
why the JSON suffix?

Reply:
It's migrated as-is, but we can rename this. Thank you